frontend/src/views/v-authorship.vue

```diff
@@ -93,15 +92,15 @@
 
   .files(v-if="isLoaded")
     .empty(v-if="files.length === 0") nothing to see here :(
-    template(v-for="(file, i) in selectedFiles")
-      .file(v-bind:key="file.path")
+    template(v-for="(file, i) in selectedFiles", v-bind:key="file.path")
+      .file
         .title
           span.path(v-on:click="toggleFileActiveProperty(file)")
             .tooltip
-              font-awesome-icon(icon="caret-down", fixed-width, v-show="file.active")
-              span.tooltip-text(v-show="file.active") Click to hide file details
-              font-awesome-icon(icon="caret-right", fixed-width, v-show="!file.active")
-              span.tooltip-text(v-show="!file.active") Click to show file details
+              font-awesome-icon(icon="caret-down", fixed-width, v-if="file.active")
```

Comment:
@gerhean I have yet to figure this issue. `v-show` does not work with `template` but it's not a template here and it works in Vue 2. I have also updated to the latest packages. I think it might just be some problem with `font-awesome-icon` itself.

Reply:
Yea upgrading incompatible packages is also part of upgrading to Vue 3